cl/phase1/network/services/batch_signature_verification.go

```diff
@@ -77,7 +82,7 @@ func (b *BatchSignatureVerifier) Start() {
 // When receiving AggregateVerificationData, we simply collect all the signature verification data
 // and verify them together - running all the final functions afterwards
 func (b *BatchSignatureVerifier) start(incoming chan *AggregateVerificationData) {
-	ticker := time.NewTicker(batchCheckInterval)
+	ticker := time.NewTicker(time.Duration(batchCheckInterval.Load()) * time.Millisecond)
```

Comment:
this is likely wrong why are you atomicizing a constant?

Reply:
yeah no keep this as normal - i am definetely not doing this just for a race warn